### PR TITLE
Document Approval on RFC-674

### PR DESCRIPTION
**Pull request by gcomoretto**

### CODE REVIEW

File: ve_details.tex [hunk 1]
```diff
@@ -0,0 +1,2698 @@
...
+  \href{https://docushare.lsst.org/docushare/dsweb/Get/DMTR-82/DMTR-82.pdf}{DMTR-82})
+\item
+  Installation of LSST DWDM and Bit Error Rate Tester (BERT) data
+  (completed 20190505
+  \href{https://docushare.lsstcorp.org/docushare/dsweb/View/Collection-7743}{collection-7743},
+  20191108
+  \href{https://docushare.lsstcorp.org/docushare/dsweb/Get/Document-35302/DAQ\%20DWDM\%20connection\%20tests\%2020191109.pptx}{DAQ
+  DWDM Connection Tests})
+\end{enumerate}
+
```
Comment (leannep) [inline ve_details.tex:355]: This is not an objective but rather a description of how the test will be carried out.  The test objective should define the overall goal and achievement of the test execution. 

E.g  "Verify the integration of the summit to base network by demonstrating a sustained and uninterrupted transfer of  data between summit and base over 1 day period at or exceeding  rates specified in
  \citeds{LDM-142}."

The phases should be moved into either the test procedure or should be separate test cases that must be executed to verify this requirement

File: ve_details.tex [hunk 2]
```diff
@@ -0,0 +1,2698 @@
...
+\end{longtable}
+
+\textbf{Objective:} \\
+After disconnecting fiber at an endpoint location on the base side of
+the ~summit - base fiber, demonstrate reconnection and recovery to
+transfer of data at or exceeding rates specified in \citeds{LDM-142} within MTTR
+specification. Network operator will provide MTTR data on links during
+commissioning and operations.\\[2\baselineskip]
+
+\textbf{Precondition:} \\
```
Comment (leannep) [inline ve_details.tex:745]: Objective reads more like a test case. Rephrase to something like: 
"Verify reliability of summit to base network by demonstrating reconnection and recovery to
transfer of data at or exceeding rates specified in \citeds{LDM-142} following a cut in network. 

Remaining details should be moved to the test case procedure.

File: ve_details.tex [hunk 3]
```diff
@@ -0,0 +1,2698 @@
...
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T193}{LVV-T193} &
+Jeff Kantor & Draft & 1 & false & Test
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+Analyze the network and show that data acquired by a DAQ can be
+transferred within the required time, i.e. verify that link is capable
+of transferring image for prompt processing in oArchiveMaxTransferTime =
+5{[}second{]}, i.e. at or exceeding rates specified in \citeds{LDM-142}.
```
Comment (leannep) [inline ve_details.tex]: Simplify to:
Verify that the base to archive network is capable
of transferring images acquired for prompt processing at or exceeding rates specified in \citeds{LDM-142}, i.e in oArchiveMaxTransferTime = 5{[}second{]}."

File: ve_details.tex [hunk 4]
```diff
@@ -0,0 +1,2698 @@
...
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T185}{LVV-T185} &
+Jeff Kantor & Draft & 1 & false & Inspection
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+Via monitoring and ~comparing with at least 6 months of historical data
+on the link, verify that the mean time between failures is less than
+summToBaseNetMTBF (90 days) over 1 year.
+
```
Comment (leannep) [inline ve_details.tex:615]: "Verify the availability of Summit to Base Network by demonstrating that the mean time between failures is less than summToBaseNetMTBF (90 days) over 1 year."

Move the details (Via monitoring and ~comparing with at least 6 months of historical data
on the link) into the test case procedure.

File: ve_details.tex [hunk 5]
```diff
@@ -0,0 +1,2698 @@
...
+\textbf{Verification Type} \\ \hline
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T195}{LVV-T195} &
+Jeff Kantor & Draft & 1 & false & Test
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+After disconnecting fiber at a location in the Base - Archive, network,
+verify recovery can occur within baseToArchNetMTTR = 48{[}hour{]}.
+
```
Comment (leannep) [inline ve_details.tex:1577]: "Verify Base to Archive Network Reliability by demonstrating that the network can recover from outages within baseToArchNetMTTR = 48{[}hour{]}."

Details about how the outage is simulated (i.e After disconnecting fiber at a location in the Base - Archive, network)  should be in the test case procedure.

File: ve_details.tex [hunk 6]
```diff
@@ -0,0 +1,2698 @@
...
+Jeff Kantor & Draft & 1 & false & Test
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+~Simulate outage on primary and verify fail over to secondary link and
+capacity on secondary link, restore primary link and verify that network
+recovers to primary. Demonstrate use of secondary path in ``catch-up''
+mode.
+
```
Comment (leannep) [inline ve_details.tex:1747]: "Verify Base to Archive Network Secondary Link failover and capacity, and subsequent recovery primary. Demonstrate use of secondary path in ``catch-up'' mode."

Details of how, e.g by simulations to go to the test case procedure.

File: ve_details.tex [hunk 7]
```diff
@@ -0,0 +1,2698 @@
...
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T200}{LVV-T200} &
+Jeff Kantor & Draft & 1 & false & Test
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+Monitor and analyze data transfers to verify networks can transfer data
+at archToDacBandwidth = 10000{[}megabit per second{]}, i.e. at or
+exceeding rates specified in \citeds{LDM-142}.
+
```
Comment (leannep) [inline ve_details.tex:1954]: Verify archiving of data to  Data Access Center Network at or exceeding rates specified in \citeds{LDM-142}, i.e at archToDacBandwidth = 10000{[}megabit per second{]},.

Montitoring/analysis details to go to the test case procedure

File: ve_details.tex [hunk 8]
```diff
@@ -0,0 +1,2698 @@
...
+\textbf{Verification Type} \\ \hline
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T201}{LVV-T201} &
+Jeff Kantor & Draft & 1 & false & Test
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+From test and historical data, extrapolate to 1 year to estimate
+networks meet archToDacNetMTBF = 180{[}day{]}.
+
```
Comment (leannep) [inline ve_details.tex:2079]: Verify availability of archiving to Data Access Center Network using test and historical data of or exceedding archToDacNetMTBF= 180{[}day{]).

File: ve_details.tex [hunk 9]
```diff
@@ -0,0 +1,2698 @@
...
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+After disconnecting fiber locations on the Archive to DACs primary path,
+verify fail-over to secondary path and capacity on secondary path.
+~After reconnecting primary path, recovery of transfer of data on
+primary path, and verify recovery can meet chToDacNetMTTR =
+48{[}hour{]}.
+
```
Comment (leannep) [inline ve_details.tex:2204]: Verify reliability of Archive to Data Access Center Network by demonstrating successful failover and capacity to the secondary part and  subsequent recovery to primary within or exceeding chToDacNetMTTR =
48{[}hour{]}.

File: ve_details.tex [hunk 10]
```diff
@@ -0,0 +1,2698 @@
...
+\textbf{Verification Type} \\ \hline
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T188}{LVV-T188} &
+Jeff Kantor & Draft & 1 & false & Inspection
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+Inspect construction and operations contracts and Indefeasible Rights to
+Use (IRUs).
+
```
Comment (leannep) [inline ve_details.tex:1228]: Verify Summit to Base Network Ownership and Operation by LSST and/or the operations entity by inspection of  operations contracts and Indefeasible Rights

File: ve_details.tex [hunk 11]
```diff
@@ -0,0 +1,2698 @@
...
+\textbf{Jira Link} & \textbf{Owner} & \textbf{Status} & \textbf{Version} & \textbf{Critical Event} &
+\textbf{Verification Type} \\ \hline
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T203}{LVV-T203} &
+Kian-Tat Lim & Draft & 1 & false & Test
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+After simulating failure on primary path, verify fail-over to and
+capacity on secondary path.
```
Comment (leannep) [inline ve_details.tex]: Verify Archive to Data Access Center Network via Secondary Link by simulating a failure on primary path and capacity on the secondary path.

File: ve_details.tex [hunk 12]
```diff
@@ -0,0 +1,2698 @@
...
+Jeff Kantor & Draft & 1 & false & Test
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+From test and historical data, extrapolate to a full year to estimate if
+expect to meet baseToArchNetMTBF = 180{[}day{]}. ~Note that this is for
+complete loss of transfer service (simultaneous failure on all paths),
+not a single path failure with successful fail-over.
+
```
Comment (leannep) [inline ve_details.tex:1452]: Verify the availability of the Base to Archive Network communications by demonstrating that it  meets or exceeds a mean time between failures, measured over a 1-yr period of   MTBF > baseToArchNetMTBF (180[day])

**Review by leannep — CHANGES_REQUESTED** [review]
Main comment is that many test case objectives contain implementation details. These details should be in the test case procedure and the objective should provide a clear statement of the goal of the test case. I have provided suggestions for rephrasing - feel free to improve on them.

**Comment by womullan** [comment]
This RFC was implemtented.. ages ago ...

**Pull request merged**